lib/board.rb

```diff
@@ -0,0 +1,66 @@
+class Board
+  attr_accessor :highest_turns
```

Comment:
make sure that you most use write/read as far as I've seen in your code you only need write